common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-allowNestedIfParentExistExt_2023-08-24-16-48.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Modify PathForTrackedResourceTypes, this rule will check the path must be under a subscription and resource group for tracked resource types only.",
```

Comment:
>Modify [](http://example.com/codeflow?start=18&length=6)

remove this changelog?

Reply:
Sorry I didnt read the change log properly. This is good

Reply:
Okay